### DISCUSSION TITLE
29 Aug. – 2 Sept.

### DISCUSSION THREAD

**Discussion by esheehan-gsl** [NOAA-GSL/unified-graphics#84] [Progress Report]
🚴🏻 **Progress**

- #81 

😖 **Regressions**

None

💬 **To Discuss**

None

📆 **Next Week**

- Implement a density plot for visualizing vector distributions

**Comment by ian-noaa** [comment]
🚴🏻 **Progress**

- Ian out on vacation. :palm_tree: 

😖 **Regressions**

None

💬 **To Discuss**

None

📆 **Next Week**

N/A
